usbms.c

```diff
@@ -987,13 +987,13 @@ int
 		}
 
 		// Wee bit of trickery with an obscure umount2 feature, to see if the mountpoint is currently busy,
-		// without actually unmounting it for real...
+		// without actually unmounting it for real…
 		rc = umount2(mount_points[i].mountpoint, MNT_EXPIRE);
 		if (rc != EXIT_SUCCESS) {
 			if (errno == EAGAIN) {
 				// That means we're good to go ;).
 				LOG(LOG_INFO,
-				    "%s storage partition wasn't busy, it's been successfully marked as expired.",
+				    "%s storage partition wasn't busy, it's been marked as expired.",
```

Comment:
NACK, it's marked expired regardless, so the distinction is important here ;).

Reply:
Explicitly?
Not really sure how it actively is marked as expired.
Reverted nonetheless, so doesn't matter.

Reply:
I misspoke, but I do like `successfully`, because it's really a marker that the action was "successful" caught inside an an error codepath, so the tricky wording makes the tricky API clearer to follow ;).

Reply:
It's also a direct reference to the manpage's wording:

`EAGAIN A call to umount2() specifying MNT_EXPIRE successfully marked an unbusy filesystem as expired.`

Reply:
Can it be unsuccessfully marked as expired, or is that an otherwise rational expectation to be avoided?
I have a similar problem with "enable" where it should be something in addition to turning something on/off.
It has been reverted though, so good to go I would think.

Reply:
The process can fail if the FS is stil busy (in which case it throws `EBUSY`), but the flag itself is atomic, it's either set successfully, or not (I'm unclear on what happens in this case, though).